cdap-app-fabric/src/main/java/io/cdap/cdap/internal/app/runtime/distributed/remote/RuntimeJobRemoteProcessController.java

```diff
@@ -69,10 +78,10 @@ public void terminate() throws Exception {
   }
 
   @Override
-  public void kill() throws Exception {
-    LOG.debug("Force stopping program run {}", programRunId);
+  public void kill(RuntimeJobStatus status) throws Exception {
+    LOG.debug("Force stopping program run {} based on program status {}", programRunId, status);
```

Comment:
remove this log line, as the underlying implementation might not do anything

Reply:
Remved.